packaging/templates/puppynet/archive/config.toml

```diff
@@ -90,10 +90,10 @@ gcmode = "archive"
     [jsonrpc.ws]
         enabled = true
         port = 8546
-        # prefix = ""
-        # host = "localhost"
-        # api = ["web3", "net"]
+        host = "0.0.0.0"
+        api = ["eth", "net", "web3", "txpool", "bor"]
```

Comment:
debug flag

Reply:
No, Should not be there by default, Generally RPCs doesn't provide debug apis

Reply:
If needed, We will add later